autogalaxy/profiles/mass/point/smbh_binary.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
+from typing import Tuple
+
+import autoarray as aa
+
+from autogalaxy.profiles.mass.abstract.abstract import MassProfile
+from autogalaxy.profiles.mass.point.smbh import SMBH
+
+
+class SMBHBinary(MassProfile):
+    def __init__(
+        self,
+        centre: Tuple[float, float] = (0.0, 0.0),
+        separation: float = 1.0,
+        angle_binary: float = 0.0,
+        mass: float = 1e10,
+        mass_ratio: float = 1.0,
+        redshift_object: float = 0.5,
+        redshift_source: float = 1.0,
+    ):
+        """
+        Represents a supermassive black hole (SMBH) binary (e.g. two merging SMBH's at the centre of a galaxy).

```

Comment:
Why is this better than simply adding two SMBHs to a model?

Reply:
Given that values computed from this object are usually just the sum of the two implicit objects it might be better to model the two objects explicitly and then compute their relative properties?

Reply:
We want to parameterize it as two objects which are separated from one another about a `centre`, with an input angle and whatnot.

If you input two SMBH's in the model, they could move anywhere over the image-plane and thus now be a binary rotating around one another.